.changes/next-release/feature-core-cced62e.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "core",
```

Comment:
nit, core changes should be `AWS SDK for Java v2`

Reply:
I see. Thanks. I was following the instructions in the README. Will change it.